### PR TITLE
PSC-19-Tabbar

### PR DESCRIPTION
**Pull request by Pryssss**

### CODE REVIEW

File: Uni/Presentation/Demo Story/ViewController.swift [hunk 1]
```diff
@@ -20,8 +20,9 @@ class ViewController: UIViewController {
     }
 
     @IBAction func launchMvpDemoAciton(_ sender: UIButton) {
-        
-        let controller = ControllersFactory.createDemoController(parametrs: (), coordinator: coordinator)
-        self.present(controller, animated: true)
+
+//        AppCoordinator.start(OnboardCoordinator.self)
```
Comment (BohdanDatskiv) [inline Uni/Presentation/Demo Story/ViewController.swift]: remove comments

File: Uni/Presentation/Demo Story/Demo/DemoPresenter.swift [hunk 2]
```diff
@@ -36,18 +36,18 @@ extension DemoPresenter: PresenterProtocol, DemoPresenterProtocol {
...
-
-        let config = AlertControllerConfig(title: AppTexts.demoAlertTitle,
-                                           message: AppTexts.demoAlertMessage,
-                                           cancel: "Cancel")
-
-        view.showAlert(config: config, options: options, title: { $0.title }, completion: { result in
-            guard let result = result else { return }
-            print(result)
-        })
+//        let options = [DemoModel(title: "Demo 1", color: AppContent.Colors.demoBlue.color),
```
Comment (BohdanDatskiv) [inline Uni/Presentation/Demo Story/Demo/DemoPresenter.swift]: remove comments

File: Uni/Resources/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -13,3 +13,9 @@
 "network_error_internal_server_down_description" = "Seems like server is tempory down, try again later";
 "demo_alert_title" = "Demo title";
 "demo_alert_message" = "Demo message";
+
+"bookmarks" = "Закладки";
```
Comment (BohdanDatskiv) [inline Uni/Resources/en.lproj/Localizable.strings]: bad naming

File: Uni/Resources/uk.lproj/Localizable.strings [hunk 4]
```diff
@@ -13,3 +13,9 @@
 "network_error_internal_server_error_description" = "Схоже сервер тимчасово недоступний, спробуйте пізніше";
 "demo_alert_title" = "Демо заголовок";
 "demo_alert_message" = "Демо повідомлення";
+
+"bookmarks" = "Bookmarks";
```
Comment (BohdanDatskiv) [inline Uni/Resources/uk.lproj/Localizable.strings]: bad naming

File: Uni/Presentation/Demo Story/Demo/DemoPresenter.swift (hunk 2 above)
Comment (Pryssss) [reply Uni/Presentation/Demo Story/Demo/DemoPresenter.swift]: POPRAVYV

File: Uni/Presentation/Demo Story/ViewController.swift (hunk 1 above)
Comment (Pryssss) [reply Uni/Presentation/Demo Story/ViewController.swift]: POPRAVYV

Comment (Pryssss) [reply Uni/Presentation/Demo Story/ViewController.swift]: ФІКСЕД

File: Uni/Resources/en.lproj/Localizable.strings (hunk 3 above)
Comment (Pryssss) [reply Uni/Resources/en.lproj/Localizable.strings]: ФІКСЕД

File: Uni/Resources/uk.lproj/Localizable.strings (hunk 4 above)
Comment (Pryssss) [reply Uni/Resources/uk.lproj/Localizable.strings]: ФІКСЕД

File: Uni/Presentation/Dashboard Story/Bookmarks Story/BookmarksPresenter.swift [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    // MARK: - Properties
+    // -------------------------------------
+    unowned var view: View!
+    weak var coordinator: Coordinator!
+    let service: Service
+
+    // -------------------------------------
+    // MARK: - Init
+    // -------------------------------------
+    required init(paramters: Paramters, service: Service, coordinator: Coordinator) {
```
Comment (RomanSlyepko) [inline Uni/Presentation/Dashboard Story/Bookmarks Story/BookmarksPresenter.swift]: typo `parameters`

File: Uni/Presentation/Dashboard Story/Bookmarks Story/BookmarksPresenter.swift [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+// -------------------------------------
+// MARK: - BookmarksPresenterProtocol, BookmarksPresenterProtocol
+// -------------------------------------
+extension BookmarksPresenter: PresenterProtocol, BookmarksPresenterProtocol {
+
+    typealias View = BookmarksViewProtocol
+    typealias Coordinator = BookmarksCoordinatorProtocol
+    typealias Paramters = Void
```
Comment (RomanSlyepko) [inline Uni/Presentation/Dashboard Story/Bookmarks Story/BookmarksPresenter.swift]: typo

File: Uni/Presentation/Dashboard Story/Map Story/MapPresenter.swift [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+// -------------------------------------
+// MARK: - MapPresenterProtocol, MapPresenterProtocol
+// -------------------------------------
+extension MapPresenter: PresenterProtocol, MapPresenterProtocol {
+
+    typealias View = MapViewProtocol
+    typealias Coordinator = MapCoordinatorProtocol
+    typealias Paramters = Void
```
Comment (RomanSlyepko) [inline Uni/Presentation/Dashboard Story/Map Story/MapPresenter.swift]: typo

File: Uni/Presentation/Dashboard Story/Settings Story/SettingsPresenter.swift [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+// -------------------------------------
+// MARK: - SettingsPresenterProtocol, SettingsPresenterProtocol
+// -------------------------------------
+extension SettingsPresenter: PresenterProtocol, SettingsPresenterProtocol {
+
+    typealias View = SettingsViewProtocol
+    typealias Coordinator = SettingsCoordinatorProtocol
+    typealias Paramters = Void
```
Comment (RomanSlyepko) [inline Uni/Presentation/Dashboard Story/Settings Story/SettingsPresenter.swift]: typo

File: Uni/Presentation/Dashboard Story/Shedule Story/SсhedulePresenter.swift [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+// -------------------------------------
+// MARK: - setting Presenter
+// -------------------------------------
+extension SchedulePresenter: PresenterProtocol, SchedulePresenterProtocol {
+
+    typealias View = ScheduleViewProtocol
+    typealias Coordinator = ScheduleCoordinatorProtocol
+    typealias Paramters = Void
```
Comment (RomanSlyepko) [inline Uni/Presentation/Dashboard Story/Shedule Story/SсhedulePresenter.swift]: typo

File: Uni/Presentation/Dashboard Story/Tasks Story/TaskPresenter.swift [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+// -------------------------------------
+// MARK: - TaskPresenterProtocol, TaskPresenterProtocol
+// -------------------------------------
+extension TaskPresenter: PresenterProtocol, TaskPresenterProtocol {
+
+    typealias View = TaskViewProtocol
+    typealias Coordinator = TaskCoordinatorProtocol
+    typealias Paramters = Void
```
Comment (RomanSlyepko) [inline Uni/Presentation/Dashboard Story/Tasks Story/TaskPresenter.swift]: typo

**Pull request merged**